CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
```

Comment:
@mssfang can you please use https://github.com/Azure/.github/blob/master/CODE_OF_CONDUCT.md for our code of conduct contents?

Also, where did the text below come from?

Reply:
I copied what Python repo is using now: https://github.com/Azure/azure-sdk-for-python/blob/master/CODE_OF_CONDUCT.md

If using the above link, we will not be consistent with Python repo's `Code of Conduct`.